src/jit/lsra.cpp

```diff
@@ -10676,20 +10743,23 @@ void LinearScan::resolveEdge(BasicBlock*      fromBlock,
         (resolveType == ResolveSharedCritical) ? REG_NA : getTempRegForResolution(fromBlock, toBlock, TYP_INT);
 #endif // !_TARGET_XARCH_
     regNumber tempRegFlt = REG_NA;
+#ifdef _TARGET_ARM_
```

Comment:
I would suggest that you not ifdef this out, if we can get away with it without the compiler(s) complaining. It will be optimized away in any case. I find this:
```
regNumber tempRegDbl = REG_NA; // Used only for ARM
```
easier on the eyes than the `#ifdef` and `#endif`

Reply:
Sure :)